src/set.rs

```diff
@@ -1135,6 +1137,26 @@ where
             None => None,
         }
     }
+
+    /// Returns a mutable reference to the [`RawTable`] used underneath [`HashSet`].
+    /// This function is only available if the `raw` feature of the crate is enabled.
+    ///
+    /// # Note
+    ///
+    /// Calling the function safe, but using raw hash table API's may require
```

Comment:
```suggestion
    /// Calling the function is safe, but using raw hash table API's may require
```
This should probably be fixed in the `HashMap` version as well.

Reply:
Fixed both places in 6041b5aef6ad990a379fd624b953c7ec26d62e18, along with a little tweaking of the grammar in that sentence. Let me know if you'd like something different!